### DISCUSSION TITLE
Show iPad's default status bar when Default Layout is set to "Fit"

### DISCUSSION THREAD

**Discussion by UrsaDK** [blinksh/blink#1813] [Ideas]
I got so used to checking the current time & date or wifi & battery status by glancing at the top left/right corners of my screen that I find not having that information there a bit disorientating. 

Is there anyway to show iPad's default status bar when Blink's "Appearance > Default Layout" is set to "Fit"?

**Comment by yury** [comment]
What if we add mode "with status bar"?

**Comment by UrsaDK** [comment]
Thanks for such a quick response @yury! From a personal point of view, that would totally work. 🙂 

Would it be possible to show the toggle only if the value of the Default Layout is set to Fit? – It probably doesn't make sense to show it if the Default Layout is set to Cover or Fill as the first line of the terminal would overlap with the status bar.

**Reply by yury** [reply]
Yep, that layout mode would be "Fit" + status bar.
We need to came up with better name for that mode.
